apps/nextra/pages/en/network/glossary.mdx

```diff
@@ -504,7 +504,7 @@ other Move scripts. It is a single-use program.
 
 To see example uses of Move scripts,
 follow [Move scripts](../build/smart-contracts/scripts/script-tutorial.mdx).
-{/* TODO: This tutorial seems to have disappeared and the [Your First Multisig](../) tutorial. */}
```

Comment:
I see the multisig tutorial in `../../build/guides/first-multisig`

Reply:
That comment was from during the migration, so does not apply anymore it seems :) 

Removed the whole comment instead of just the link.